metadata.rb

```diff
@@ -0,0 +1,13 @@
+name 'centreon-engine'
+maintainer 'Criteo'
+maintainer_email 'c.judith@criteo.com'
+license 'all_rights'
+description 'Installs/Configures centreon-engine'
+long_description 'Installs/Configures centreon-engine'
+version 	 '0.1.0'
+depends          'sudo'
+depends          'postfix'
+depends		 'yum', '~> 3.2'
```

Comment:
also don't use pessimistic constraint, it is really a pain to integrate cookbook with pessimistic constraints